linkerd/telemeter/usage/src/main/scala/io/buoyant/linkerd/telemeter/UsageDataTelemeter.scala

```diff
@@ -79,6 +84,13 @@ private[telemeter] object UsageDataTelemeter {
   def mkContainerManager: Option[String] =
     sys.env.get("MESOS_TASK_ID").map(_ => "mesos")
 
+  def mkStartTime: Option[String] = {
+    val tz = TimeZone.getTimeZone("UTC")
+    val df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'")
+    df.setTimeZone(tz)
+    Some(df.format(new Date()))
```

Comment:
Good idea. Thinking about it more, I'd rather send uptime. I'll update the proto accordingly